core/state_transition_ovm.go

```diff
@@ -0,0 +1,196 @@
+package core
+
+import (
+	"bytes"
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/core/vm"
+)
+
+var GodAddress = common.HexToAddress("0x444400000000000000000000000000000000000")
+var ZeroAddress = common.HexToAddress("0x0000000000000000000000000000000000000000")
+
+type ovmTransaction struct {
+	Timestamp     *big.Int       "json:\"timestamp\""
```

Comment:
Why the escaped `"`?

Reply:
Took this from another part of geth where it used the escaped `"`.